pkg/stack/loader.go

```diff
@@ -77,7 +77,7 @@ func LoadStacks(stackService Service) {
 }
 
 func extractRequiredParameters(file []byte) []string {
-	regexStr := "\\{\\{[ ]?requiredEnv[ ]?\"(.*?)\"[ ]?\\}\\}"
+	regexStr := "\\{\\{[ ]?requiredEnv[ ]?\"(.*?)\".*\\}\\}"
```

Comment:
does that relate to switching from ids to names?

Reply:
then what is it 😋 its easier to review a PR if it only contains whats in the PR title/description/task.